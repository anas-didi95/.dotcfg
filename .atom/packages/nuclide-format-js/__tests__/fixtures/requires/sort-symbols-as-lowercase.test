const Zaz = require('Zaz');

const _ = require('underscore');
const $ = require('jQuery');
const foo = require('foo');

const r = _($(foo(Zaz)));
