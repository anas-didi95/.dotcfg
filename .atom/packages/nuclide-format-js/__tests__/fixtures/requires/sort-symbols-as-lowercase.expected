const Zaz = require('Zaz');

const foo = require('foo');
const $ = require('jQuery');
const _ = require('underscore');

const r = _($(foo(Zaz)));
